code/datums/status_effects/debuffs/cursed.dm

```diff
@@ -0,0 +1,187 @@
+			messages += span_boldwarning("As you shut your eyes to dwell on this conundrum, the brand surges in pain. You shudder to think what might happen if you go unconscious.")
+
+		if(3)
+			owner.emote("cough")
+			messages += span_boldwarning("Your throat becomes to feel like it's slowly caking up with sand and dust. You eject the contents of the back of your throat onto your good hand.")
+			messages += span_boldwarning("It is sand. Crimson red. You've never felt so thirsty in your life, yet you don't trust your own hand to carry the glass to your lips.")
+			messages += span_boldwarning("You get the sneaking feeling that if someone else were to win, that it might clear your curse too. Saving your life is a noble cause.")
+			messages += span_boldwarning("Of course, you might have to not speak on the nature of this machine, in case they scamper off to leave you to die.")
+			messages += span_boldwarning("Is it truly worth it to condemn someone to this fate to cure the manifestation of your own hedonistic urges? You'll have to decide quickly.")
+
+		if(4)
+			messages += span_boldwarning("A migraine swells over your head as your thoughts become hazy. Your hand desperately inches closer towards the slot machine for one final pull...")
+			messages += span_boldwarning("The ultimate test of mind over matter. You can jerk your own muscle back in order to prevent a terrible fate, but your life already is worth so little now.")
+			messages += span_boldwarning("This is what they want, is it not? To witness your failure against itself? The compulsion carries you forward as a sinking feeling of dread fills your stomach.")
+			messages += span_boldwarning("Paradoxically, where there is hopelessness, there is elation. Elation at the fact that there's still enough power in you for one more pull.")
+			messages += span_boldwarning("Your legs desperate wish to jolt away on the thought of running away from this wretched machination, but your own arm remains complacent in the thought of seeing spinning wheels.")
+			messages += span_userdanger("The toll has already been exacted. There is no longer death on 'your' terms. Is your dignity worth more than your life?")
+
+		if(5 to INFINITY)
+			if(max_curse_count != DEFAULT_MAX_CURSE_COUNT) // this probably will only happen through admin schenanigans letting people stack up infinite curses or something
+				to_chat(owner, span_boldwarning("Do you <i>still</i> think you're in control?"))
+				return
+
+			to_chat(owner, span_userdanger("Why couldn't I get one more try?!"))
+			owner.investigate_log("has been gibbed by the cursed status effect after accumulating [curse_count] curses.", INVESTIGATE_DEATHS)
+			owner.gib()
+			qdel(src)
+			return
+
+	for(var/message in messages)
+		to_chat(owner, message)
+		sleep(1.5 SECONDS) // yes yes a bit fast but it can be a lot of text and i want the whole thing to send before the cooldown on the slot machine might expire
```

Comment:
Do we need to worry about them playing it while this is sleeping? 

In a similar vein you may want a sanity check for if the effect is qdel's mid monologue because it'll runtime when owner is set to null

Reply:
no because we cooldown for 15 seconds and this shouldn't exceed 10 seconds. i'll do the qdeleted check just in case though

Reply:
also `to_chat()` nullchecks the target anyways, i don't get what would runtime

![image](https://github.com/tgstation/tgstation/assets/34697715/47ba92a8-e7af-4e0c-b13e-7216a340341b)

Reply:
i would use addtimer to_chat instead